jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/requestlist/RequestListViewImpl.java

```diff
@@ -148,82 +145,65 @@ public void initColumns( ExtendedPagedTable extendedPagedTable ) {
 
     @Override
     public void initSelectionModel(ExtendedPagedTable<RequestSummary> extendedPagedTable) {
-        extendedPagedTable.setEmptyTableCaption( constants.No_Jobs_Found() );
+        extendedPagedTable.setEmptyTableCaption(constants.No_Jobs_Found());
     }
 
     private Column<RequestSummary, RequestSummary> initActionsColumn() {
         List<HasCell<RequestSummary, ?>> cells = new LinkedList<HasCell<RequestSummary, ?>>();
 
         cells.add(new ActionHasCell(constants.Details(),
                                     job -> presenter.showJobDetails(job),
-                                    RequestStatus.QUEUED,
-                                    RequestStatus.DONE,
-                                    RequestStatus.CANCELLED,
-                                    RequestStatus.ERROR,
-                                    RequestStatus.RETRYING,
-                                    RequestStatus.RUNNING));
+                                    presenter.getDetailsActionStates()));
```

Comment:
Instead of asking the presenter for status, I would suggest making it standard that we get the predicate (getCondition) only. That way we could also remove `ActionHasCell`. The logic if a specific status match or not, would then be done in the presenter.